config/state.ts

```diff
@@ -79,6 +80,27 @@ const initialState: State = {
 	showInstructions: true,
 	darkMode: true,
 	enableSFXConfetti: true,
+	enableSound: true,
+	sounds: {
+		type: [
+			'/sounds/type.wav',
+		],
+		wordComplete: [
+			'/sounds/word-complete.wav',
+		],
+		gameComplete: [
+			'/sounds/game-complete.wav',
+		],
+		streakComplete: [
+			'/sounds/streak-complete.wav',
+		],
+		failureSlow: [
+			'/sounds/failure-slow.wav',
+		],
+		failureTypo: [
+			'/sounds/failure-typo.wav',
+		],
+	},
```

Comment:
There is only a single entry in each array, so let's collapse them down as it's a bit neater.

```suggestion
	sounds: {
		type: ['/sounds/type.wav'],
		wordComplete: ['/sounds/word-complete.wav'],
		gameComplete: ['/sounds/game-complete.wav'],
		streakComplete: ['/sounds/streak-complete.wav'],
		failureSlow: ['/sounds/failure-slow.wav'],
		failureTypo: ['/sounds/failure-typo.wav'],
	},
```